framework/src/problems/FEProblemBase.C

```diff
@@ -2619,6 +2664,48 @@ FEProblemBase::reinitMaterialsNeighbor(SubdomainID blk_id, THREAD_ID tid, bool s
   }
 }
 
+void
+FEProblemBase::reinitMaterialsNeighborOnInterface(BoundaryID boundary_id,
```

Comment:
I'm not sure yet if we need this. 
How do I reinit neighbor materials starting from a boundary id?